plugins/coatils.py

```diff
@@ -163,7 +165,9 @@ def position(stl, stc, enl, enc):
                                                               enl, enc)
 
     # Ignore PycodestyleBear, LineLengthBear
-    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$')
+    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$',
+               re_cmd_name_help='run <Bear [[setting=value]+]>+\n'
+                                '```\n<code>+\n```')
```

Comment:
I'd remove the `<>` to avoid any confusion with html tags

Also, even someone experienced using the bot with help text `[setting=value]+]>+` would fail to decipher it